packages/browser-wallet/src/popup/pages/Account/SendCcd/ConfirmTransfer.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useAtomValue } from 'jotai';
+import { selectedAccountAtom } from '@popup/store/account';
+import { AccountAddress, AccountTransactionType, SimpleTransferPayload, TransactionExpiry } from '@concordium/web-sdk';
+import { sendTransaction } from '@popup/shared/utils/transaction-helpers';
+import { jsonRpcClientAtom } from '@popup/store/settings';
+import { usePrivateKey } from '@popup/shared/utils/account-helpers';
+import Button from '@popup/shared/Button';
+import DisplaySimpleTransfer from '@popup/shared/TransactionReceipt/displayPayload/DisplaySimpleTransfer';
+import { useNavigate } from 'react-router-dom';
+import { absoluteRoutes } from '@popup/constants/routes';
+import TransactionReceipt from '@popup/shared/TransactionReceipt/TransactionReceipt';
+import { routes } from './routes';
+
+interface Props {
+    payload: SimpleTransferPayload | undefined;
```

Comment:
Why would the payload ever be `undefined` when this page is shown? In that case there would be nothing to confirm.

Reply:
It wouldn't, it was just "lazy" typing. I have refactored this away though.